build.sbt

```diff
@@ -40,8 +40,8 @@ def getPublishTo(snapshot: Boolean) = {
 
 lazy val commonSettings = Seq(
   organization:= "com.stripe",
-  scalaVersion := "2.13.1",
-  crossScalaVersions := List(scalaVersion.value, "2.12.10", "2.11.12"),
+  scalaVersion := "2.12.10",
```

Comment:
This was to make Metals happier, which otherwise was trying to build the plotting code under 2.13

Reply:
Oh nice, I was encountering this as well.